src/Test/Monad/Morph.hs

```diff
@@ -10,12 +10,20 @@ import Test.QuickCheck.HigherOrder (Equation(..))
 -- | Natural transformation.
 type m ~> n = forall t. m t -> n t
 
+-- |
+-- @
+-- hom (m '>>=' k) = hom m '>>=' hom . k
+-- @
 bindHom
   :: forall m n a b
   .  (Monad m, Monad n)
   => (m ~> n) -> m a -> (a -> m b) -> Equation (n b)
 bindHom hom m k = hom (m >>= k) :=: (hom m >>= hom . k)
 
+-- |
+-- @
+-- hom ('return' a) = 'return' a
```

Comment:
Same as before: something like 'Combining `return` with a natural transformation is the same as using `return` directly.', maybe?